api/bazel/repositories.bzl

```diff
@@ -34,6 +34,16 @@ def api_dependencies():
         locations = REPOSITORY_LOCATIONS,
         build_file_content = KAFKASOURCE_BUILD_CONTENT,
     )
+    envoy_http_archive(
+        name = "kafka_server_binary",
+        locations = REPOSITORY_LOCATIONS,
+        build_file_content = KAFKASERVERBINARY_BUILD_CONTENT,
+    )
+    envoy_http_archive(
+        name = "kafka_python_client",
```

Comment:
Why do these deps need to come in via the API deps vs the regular Envoy ones? I think the APIs don't need either of these.

Reply:
I'm not too familiar with bazel, but the previous Kafka-related dependency was added here, so I did the same.
If these 2 (or 3 even) should be migrated somewhere else, please let me know - happy to move.

Reply:
I think they should be moved to https://github.com/envoyproxy/envoy/blob/master/bazel/repositories.bzl? I'm not sure why the other ones are part of the API bazel requirements. It seems like that is/was a mistake?